misc/MajorityVoting.ipynb

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
<p>Hard coded paths. Oof. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I was younger and naive and hopeful version of myself back then. Will change :P</p>